src/components/PaginationFooter/Buttons/LastPage.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Button from './Button';
+import { usePaginationContext } from '../context/PaginationProvider';
+import handleLastTab from '../../../utils/handle-last-tab';
+
+const LastPage = () => {
+  const { pageInfo, keyboard, totalPages, isSelectionMode, width } = usePaginationContext();
+  const onLastPage = pageInfo.page >= totalPages - 1;
+
+  if (width <= 350) {
```

Comment:
magic number, i would prefer to read this from constants directory with a name like `PAGINATION_BUTTON_VISIBILITY_TRESHOLD` or some other preferable name

Reply:
Also maybe it's better to move showing/not-showing responsibility on the parent component of these buttons since the width comes from `usePaginationContext` and it's all for same four types of buttons. 

I also notice that you took this approach to make `PaginationContext` a clean module, but maybe it's a bit much work in this case to not handle all logic in one place and spread through multiple components IMO